### ISSUE TITLE
New term - preservationMethod

### ISSUE THREAD

**Issue by mdoering** [tdwg/dwc#1]
Looking into the GBIF data it appears that dwc:preparations is used to capture 2 distinct pieces of information, the preservation method and the part of the organism being stored.

A new term "preservationMethod" is requested to capture the one part. NCD has created a vocabulary for preservation methods in the TDWG ontology that could be used as a recommended vocabulary: http://rs.tdwg.org/ontology/voc/Collection#SpecimenPreservationMethodTypeTerm
#### preservationMethod definition

> The process or technique used to prevent physical deterioration of non-living collections.
> Recommended best practice is to use a controlled vocabulary such as the NCD Specimen Preservation Method Type Term vocabulary.
#### Examples

dried; driedAndPressed; fluidPreserved; frozen; pinned

**Comment by tucotuco** [comment]
Andy Bentley is leading a group to determine how best to capture information about collection objects, and their preservation methods and histories.

**Comment by tucotuco** [comment]
This proposal requires a new term change request using the template in the [Guidelines for contributing] (https://github.com/tdwg/dwc/blob/master/.github/CONTRIBUTING.md). there has been demand from multiple sources, though not captured in this issue. Documented demand would be useful.

**Comment by acbentley** [comment]
I had proposed a preparation extension to Darwin Core to capture the varied information currently captured in the preparation field.  It is usually a concatenation of various pieces of information and is more than just the preservation method and part of the organism.  In some cases (for lot based collections), it also contains a count as well as who prepared it, date of preparation, original fixation, historic preservation, etc.  I am afraid I never made much headway with this (kept falling to the bottom of my to-do list) but would be interested in reinvigorating it if others are interested.  Attached is an Excel file that we worked on with proposed terms, and a PPT presentation I gave as part of Darwin Core hour
[Preparation extension field list.xlsx](https://github.com/tdwg/dwc/files/5198536/Preparation.extension.field.list.xlsx)
[Darwin Core Hour 04a - Evolution of Darwin Core Terms and Extensions_ two extant examples for community input.pptx](https://github.com/tdwg/dwc/files/5198541/Darwin.Core.Hour.04a.-.Evolution.of.Darwin.Core.Terms.and.Extensions_.two.extant.examples.for.community.input.pptx).  Happy to discuss further with interested parties.

**Comment by mdoering** [comment]
It makes sense to have an extension to capture complex preparation details. But I would still argue that DwC should provide at least 2 separate terms for what [organism part](https://github.com/tdwg/dwc/issues/3) has been collected/stored and how it was prepared so this can be shared in a simple, flat view. Both are vital information for users.

**Comment by tucotuco** [comment]
I agree with the need for two terms. I was just hoping to keep preparations
for the organism part part. ;-)  Maybe it would be a lot easier to coin the
two new terms after all and leave preparations as it is - a sort of
verbatim field that doesn't try to have a controlled vocabulary.

**Comment by tmcelrath** [comment]
I agree with @tucotuco that this needs to be split.

**Comment by tucotuco** [comment]
Further support provided through GBIF feedback gbif/portal-feedback#2429.

**Comment by tucotuco** [comment]
I think that, along with Issue #3, these proposals would benefit from a Task Group.

**Comment by tucotuco** [comment]
Related issues are Issue #3, Issue #24 (reopened because of renewed interest), Issue #314, Issue #332, Issue #344, Issue #345, Issue #346, and Issue #347.
